tasks/reporters/Dots/index.js

```diff
@@ -0,0 +1,80 @@
+module.exports = function(grunt) {
+	var icons = {
+		error: '⁂ ',
+		info: '*',
+		warning: '⁑'
+	}
+
+	var counters = {
+		error: 0,
+		info: 0,
+		total: 0,
+		warning: 0,
+		complex: 0,
+		maintain: 0
+        }
+
+	var DotsReporter = function(filenames, options) {
+		this.options = options;
+	};
+
+	DotsReporter.prototype = {
+
+		colors: {
+			info: 'grey',
+			warning: 'yellow',
+			error: 'red'
+		},
+
+		violations: function(filepath, violations) {
+			violations.forEach(function(data) {
+				counters[data.severity]++;
+				counters.total++;
+				if(data.complex) {
+					counters.complex++;
+				} else if (data.unmaintainable) {
+					counters.maintain++;
+				}
+				this.log(icons[data.severity][this.colors[data.severity]]);
+			}, this);
+		},
+
+		start: function() {
+			grunt.log.writeln('======================================================================'.green);
+			grunt.log.writeln(' Complexity report '.green);
+			grunt.log.writeln('——————————————————————————————————————————————————————————————————————'.green);
+			counters.error = 0;
+			counters.info = 0;
+			counters.total = 0;
+			counters.warning = 0;
+			counters.complex = 0;
+			counters.maintain = 0;
+		},
+
+		finish: function() {
+			grunt.log.writeln(' ');
+			grunt.log.writeln('——————————————————————————————————————————————————————————————————————'.green);
+			grunt.log.write  (' Low: '.green + String('   ' + counters.info).slice(-3).grey);
```

Comment:
It would prob be better to stick to the same naming convention - info, warning and error rather than low, high, critical. I know PMD goes against that, but I think that should be the exception